modules/tinymce/src/plugins/lists/main/ts/ui/MenuItems.ts

```diff
@@ -17,7 +17,7 @@ const register = (editor: Editor): void => {
     text: 'List properties...',
     icon: 'ordered-list',
     onAction: () => editor.execCommand('mceListProps'),
-    onSetup: (api) => Util.listState(editor, 'OL', (active) => api.setDisabled(!active))
+    onSetup: (api) => Util.listState(editor, 'OL', (active) => api.setEnabled(active))
```

Comment:
This can be ETA reduce now with this new API:
```suggestion
    onSetup: (api) => Util.listState(editor, 'OL', api.setEnabled)
```